api/organisations/chargebee/chargebee.py

```diff
@@ -177,6 +186,14 @@ def add_single_seat(subscription_id: str):
         )
 
     except ChargebeeAPIError as e:
+        api_error_code = e.json_obj["api_error_code"]
+        if api_error_code in CHARGEBEE_PAYMENT_ERROR_CODES:
+            logger.warning(
+                "Payment declined during additional seat upgrade to a "
+                f"CB subscription for subscription_id {subscription_id}"
+            )
```

Comment:
I wasn't sure whether we wanted to log the specific payment failure here or not.

Reply:
I think probably best to include it, yep.

Reply:
I have added this so we can get this merged 👍

Reply:
Thank you!!